src/test/java/com/microsoft/sqlserver/jdbc/RandomUtil.java

```diff
@@ -41,7 +41,7 @@ static public String getIdentifier(String prefix, int maxLength, boolean unique,
         sb.append(prefix);
         sb.append("_");
         sb.append("jdbc_");
-        sb.append(System.getProperty("user.name"));
+        sb.append(System.getProperty("user.name").replace(".", "_"));

```

Comment:
there are other characters which are not valid characters if we are to strip 1 then should get rid of all. Alternatively could just get rid of this altogether